zephyr-apps/overlays/custom-pin.overlay

```diff
@@ -0,0 +1,6 @@
+#include <zephyr/dt-bindings/gpio/gpio.h>
```

Comment:
For overlay files this is how we will want to manage them:

They should be board specific - since the configuration and specific pins used will be different for different boards, so rename to `<board_name>.overlay`